conda_build/cli/main_cookie.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import sys
+import conda_build.api as api
+from attrdict import AttrDict
+from conda_build.config import Config
+
+
+def parse_args(args):
+    p = argparse.ArgumentParser()
```

Comment:
Conda build's commands all extend conda. This is good and bad. It makes the system nice and uniform, but it is also very slow. Until we do something else, we should stick to the way things are done now.  However, I'm not at all clear on why this whole PR should be a new conda build feature and not just another cookiecutter recipe that we can point to.